tests/unit_tests/conftest.py

```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import pytest
+
+from superset.app import create_app
```

Comment:
please put the imports inside the fixture since loading superset will do a lot of unnecessary loadings

Reply:
Ah, good catch. Will fix it. But keep in mind that just importing `create_app` doesn't do any initialization, and the import is reasonably quick.

Reply:
Oh, for sure. I was talking about how long it used to take before we moved things into `create_app` — then it would take many seconds to import anything from Superset.